### PR TITLE
Document the assistant logic a bit more clearly

### PR DESCRIPTION
**Pull request by ShiftyRail**
Did part of that code, it's confusing, bad, and there's a seemingly arbitrary -1 in it that might fool people into thinking it's a bug/oversight. 

It's not, so I'm adding some needed comment to a gunked file.

### CODE REVIEW

File: code/game/jobs/job_controller.dm [hunk 1]
```diff
@@ -1,5 +1,10 @@
 var/global/datum/controller/occupations/job_master
 
+#define FREE_ASSISTANTS 2
+
+// The logic requires a shift of 1. The technical reason is that the way it is written, it boils to if (0 > 0) {"reject the assistants"}. Unfortunately, 0 is not > 0.
```
Comment (SonixApache) [inline code/game/jobs/job_controller.dm:5]: // The logic requires a shift of 1. The technical reason is that with the way the system is written, it boils down to if (0 > 0) {"reject the assistants"}. Unfortunately, 0 is not > 0.

Missed some words for clarity. This still confuses me (how is -1 > 0 any better?) but I'm sure it's fine.

**Comment by SonixApache** [comment]
Now, I may be a brainlet, but this doesn't really clarify anything to me. In fact, it might just be making it more confusing.

**Comment by PrimeDSS13** [comment]
change the free assistant define to change how many assistants are open roundstart

**Comment by Exxion** [comment]
Couldn't you just change all these `>` to `>=` and use `FREE_ASSISTANTS` instead of `FREE_ASSISTANTS_BRUT`?

EDIT: Oh, I see, the second number isn't guaranteed to be an integer, so no
I still feel like there must be a way to make this clearer

**Comment by Kurfursten** [comment]
I don't see how this is is at all clearer but it's functionally the same so whatever. You can always PR again.

**Pull request merged**